src/pages/task/actionButtons/relevantCommits/index.tsx

```diff
@@ -156,6 +185,19 @@ export const PreviousCommits: React.FC<PreviousCommitsProps> = ({ taskId }) => {
       >
         Go to {versionMetadata?.isPatch ? "base" : "previous"} commit
       </MenuItem>
+      <MenuItem
+        as={Link}
+        disabled={breakingLoading || breakingTask === undefined}
```

Comment:
This is more of a UX comment, but when testing this on larger projects (like mongodb), it can take a while for the query to finish. I think it could be good to have some indication that the query is still loading (maybe a loading spinner)?

You could make a separate ticket to explore this, since it wasn't covered by the original designs! Or you can experiment (but only if you want to)!

Reply:
Hm that might be a good idea. I think adding a loading indicator to the relevant task button might be useful but from my testing, usually it loads as fast as the main page data. I think it would be nice if we did think of some standard for links/buttons/sections that are in the process of loading but we _have_ sent a query for them.

I thought it could be a less grayed out button, italicized, some small icon next to it, or a loading spinner next to it. I just tried some with like italicized but I'm not confident in how they look so I think I'll leave it up for future discussion